.buildkite/setup_al2.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -eu
+
+source ./.buildkite/al2env.sh
+
+echo "Creating necessary directories"
+
+mkdir -p $dir
+mkdir -p $bin_path
+mkdir -p $devmapper_path
+mkdir -p $state_path
+
+echo "Resetting thinpool $uniqueID"
+./tools/thinpool.sh reset $uniqueID
+
+export INSTALLROOT=$dir
+export FIRECRACKER_CONTAINERD_RUNTIME_DIR=$dir
+echo "Running make"
+make
+echo "Running make install"
+sudo -E "INSTALLROOT=$INSTALLROOT" make install
```

Comment:
Looks like [it can't find the go binary](https://buildkite.com/firecracker-microvm/firecracker-containerd-al2/builds/29#0c1d96de-cd1f-4ffd-903a-14ab8defe11c/50-94).

Reply:
Fixed